extras/linked-token/README.md

```diff
@@ -1,66 +1,200 @@
-## Foundry
+# Linked Token Readme
 
-**Foundry is a blazing fast, portable and modular toolkit for Ethereum application development written in Rust.**
+## Outline for Linked Token Bridge
+- Contracts
+    - ERC20 token named USDCTest on Calibration Network
+    - Token Controller contract on Calibration Network
+    - Token Replica contract on subnet
```

Comment:
This can be used in any set of networks, including local testnets, and mainnet roots. The USDCTest contract provided here is just for testing purposes, but is in no way dependent on Calibration either.